mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -2712,6 +2712,9 @@ struct PadOpVectorizationWithInsertSlicePattern
 
 void mlir::linalg::populatePadOpVectorizationPatterns(
     RewritePatternSet &patterns, PatternBenefit baseBenefit) {
+  // TODO: The following pattern implements "decomposition" and
```

Comment:
nit. formatting seems wrong. clang-format?

Reply:
This is meant to match the code below rather than the indented arguments above :)